### ISSUE TITLE
404 downloading Qt

### ISSUE THREAD

**Issue by rodrigopitanga** [Ultimaker/cura-build#100]
Apparently Qt 5.5 isn't available from download.qt.io anymore:

```
[  8%] Performing download step (download, verify and extract) for 'Qt'
-- downloading...
       src='http://download.qt.io/official_releases/qt/5.5/5.5.1/single/qt-everywhere-opensource-src-5.5.1.tar.gz'
       dst='/home/rodrigo/devel/cura-build/build/Qt-prefix/src/qt-everywhere-opensource-src-5.5.1.tar.gz'
       timeout='none'
error: downloading 'http://download.qt.io/official_releases/qt/5.5/5.5.1/single/qt-everywhere-opensource-src-5.5.1.tar.gz' failed
  status_code: 22
  status_string: "HTTP response code said error"
  log:
  --- LOG BEGIN ---
    Trying 77.86.229.90...
TCP_NODELAY set
Connected to download.qt.io (77.86.229.90) port 80 (#0)
GET /official_releases/qt/5.5/5.5.1/single/qt-everywhere-opensource-src-5.5.1.tar.gz HTTP/1.1
Host: download.qt.io
User-Agent: curl/7.51.0
Accept: */*

The requested URL returned error: 404 Not Found
Curl_http_done: called premature == 1
Closing connection 0

  --- LOG END ---
```
Qt 5.6 and 5.7 are available, though.

Edit: Qt 5.5.1 is in archived downloads and probably unmaintained.

**Comment by nickthetait** [comment]
See https://code.alephobjects.com/rCB864be11835361c73578e8ecc37d9ca1439f25367

**Comment by Ghostkeeper** [comment]
Updated the link. I did it on a branch though, due to the state of the current release.

**Comment by rodrigopitanga** [comment]
Works for me now

**Comment by LipuFei** [comment]
Fixed by #101, @rodrigopitanga I will close this issue.
